libr/util/pj.c

```diff
@@ -3,6 +3,8 @@
 #include <r_util.h>
 #include <r_util/r_print.h>
 
+static int pj_encoding = PJ_ENCODING_DEFAULT;
```

Comment:
Globals are very bad, in particular when used inside a utility library like `pj`.

Reply:
Ok, so we're saying that we need to make an additional constructor (e.g. `pj_new_with_encoding `) and in locations where we deem it necessary to encode, we create the `pj` instance using that? 

Just means we're going to have a lot of places that fetch the value from the config and then update it inside pj.
Will make the adjustment (I originally did this before pushing but it felt really strained) and then let you all decide where we want `pj_new` to be swapped with `pj_new_with_encoding`

Reply:
Adjusted.